crates/driver/src/infra/config/file/mod.rs

```diff
@@ -63,10 +63,19 @@ struct Config {
     )]
     order_priority_strategies: Vec<OrderPriorityStrategy>,
 
+    /// The maximum number of `/settle` requests that can be queued up
+    /// before the driver starts dropping new requests.
+    #[serde(default = "default_settle_queue_size")]
+    settle_queue_size: usize,
+
     /// Archive node URL used to index CoW AMM
     archive_node_url: Option<Url>,
 }
 
+fn default_settle_queue_size() -> usize {
+    3
```

Comment:
Not sure if it makes sense to keep more than 3 pending settlements, at least for mainnet.